libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -211,6 +211,7 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @Values{Plane}: 108:QRTL Mode
     // @Values{Copter}: 109:use Custom Controller
     // @Values{Copter, Rover, Plane, Blimp, Sub}:  110:KillIMU3
+    // @Values{Copter, Rover, Plane, Blimp, Sub}:  111:Loweheiser starter
```

Comment:
Sub and Blimp seem awfully unlikely :-)

Reply:
true, but there should be compilation guards added in that case, correct?this reflects current code capabilities

Reply:
Check, this is the correct value.